notes/06-unit-testing/figures/04-testDatabase.php

```diff
@@ -9,7 +9,7 @@ public function testDatabase()
     ]);
 
     // get the first article back from the database
-    $articleFromDB = Article::all()->first();
+    $articleFromDB = Article::first();
```

Comment:
I feel like there was a reason I'd used `all()->first()` in the DB tests.

I think maybe I'd realised you couldn't use `find(1)` if you're using MySQL for testing (as the IDs don't get reset). And then just brain farted to `all()->first()` instead of `first()`).